plugins/usi-dock/fu-usi-dock-mcu-device.c

```diff
@@ -729,26 +746,19 @@ fu_usi_dock_mcu_device_cleanup(FuDevice *device,
 {
 	g_autoptr(FwupdRequest) request = fwupd_request_new();
 
+	/* wait for the user to unplug then start the 40 second timer */
+	fu_device_add_private_flag(device, FU_USI_DOCK_DEVICE_FLAG_WAITING_FOR_UNPLUG);
+	fu_device_set_remove_delay(device, 900000);
```

Comment:
The comment says 40s..

Reply:
ohh, it's 40 seconds before we ask the user to plug the cable back in -- but it's much longer before the device comes back. The whole update takes over 15 minutes!

Reply:
!!!!

Reply:
Yup, not ideal -- but USI have some ideas on how to speed it up.